src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareFunction(a, b) {
+    const aStr = String(a);
+    const bStr = String(b);
+
+    if (aStr > bStr) {
+      return 1;
+    }
+
+    if (aStr < bStr) {
+      return -1;
+    }
+
+    return 0;
+  }
+
+  [].__proto__.sort2 = function (customCompareFunction = compareFunction) {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = 0; j < this.length - 1 - i; j++) {
+        if (customCompareFunction(this[j], this[j + 1]) > 0) {
+          [this[j], this[j + 1]] = [this[j + 1], this[j]];
```

Comment:
The swap operation using array destructuring is neat and modern. Just be aware that this approach might be slightly less performant than using a temporary variable due to the creation of temporary arrays during destructuring. However, for modern JavaScript engines, this difference is negligible.